tests/integration/test_wazuh_db/test_configuration/data/config_templates/global/config_wazuhdb_getconfig.yaml

```diff
@@ -0,0 +1,41 @@
+-
+  name: Get config missing parameter
+  test_case:
+    -
+      input: '{"version": 1, "origin": {"module": "api"}, "command": "getconfig", "module": "api"}'
+      output: '{"error":4,"message":"Empty parameters","data":{}}'
+
```

Comment:
Additional line break, could be removed to preserve the spacing of the rest of the file.